### DISCUSSION TITLE
Can't add Apple TV to Homekit

### DISCUSSION THREAD

**Discussion by anqdc** [maxileith/homebridge-appletv-enhanced#209] [General]
Hello,

I installed this plugin a few months ago and it was working great! I recently noticed however that it had stopped working... So I went to inspect the problem and couldn't really figure anything out. So I decided my best course of action was just to delete the whole thing and start from scratch. I was able to pair the Apple TV to Homebridge once again, but the Apple TV does not appear as a device in my Home app, I can add the Apple TV Enhanced Bridge, but that changes nothing, when I type the code in, it just brings me back to the "Select an accessory - My accessory isn't shown here" prompt.

I've been at it for a few hours and I've tried everything I can think of...

Can anybody help me?

Cheers,
Albert

**Comment by maxileith** [comment]
Have you unpaired the Apple TVs on your homebridge before attempting to repair?

![image](https://github.com/maxileith/homebridge-appletv-enhanced/assets/44269066/614474e0-2dac-4801-90d7-4f51f6bb01b9)

**Reply by anqdc** [reply]
Well that was ridiculously easy... 😂 Thanks for your quick response and help, it works perfectly again now! 👌🏼

Cheers,
Albert ✌🏼
